app/assets/javascripts/events.js

```diff
@@ -3,14 +3,28 @@
 
 jQuery(function() {
 
-    $('#send-emails-modal').on('show.bs.modal', function (event) {
-        var button = $(event.relatedTarget);
-        var header = button.data('title');
-        var list = button.data('list');
-        var modal = $(this);
-        modal.find('.modal-title').text(header);
-        modal.find('#send-emails-mailto').attr('href', 'mailto:' + list);
-        modal.find('#send-emails-list').val(list);
+  $('#send-emails-modal').on('show.bs.modal', function (event) {
+    var button = $(event.relatedTarget);
+    var header = button.data('title');
+    var list = button.data('list');
+    var modal = $(this);
+    modal.find('.modal-title').text(header);
+    modal.find('#send-emails-mailto').attr('href', 'mailto:' + list);
+    modal.find('#send-emails-list').val(list);
+  });
+
+  // work around so that we can have a multiline placeholder
+  $("#description")
+    .val(function() { return PLACEHOLDER; }).css('color', '#bbb')
```

Comment:
This part would override existing content in the text field. You could use
```
.val(function(index, oldValue) { return oldValue || PLACEHOLDER; });
```
to fix this.

Additionally, jQuery() onload won't trigger for turbolinks navigation. You could try if turbolinks-jquery will fix that or alternatively use the same logic as found in `app/assets/javascripts/bootstrap.js` to make sure the code is triggered on turbolink loads as well.